src/Storage/RecordModifier.php

```diff
@@ -109,16 +109,16 @@ private function setPostedValues(Content $content, $formValues, $contentType)
         $formValues = Input::cleanPostedData($formValues);
         unset($formValues['contenttype']);
 
+        $user = $this->app['users']->getCurrentUser();
         if ($id = $content->getId()) {
             // Owner is set explicitly, is current user is allowed to do this?
             if (isset($formValues['ownerid']) && (integer) $formValues['ownerid'] !== $content->getOwnerid()) {
-                if (!$this->app['permissions']->isAllowed("contenttype:{$contentType['slug']}:change-ownership:$id")) {
```

Comment:
I can… But I don't know that I made a complete accurate representation of our discussion in my comment, ergo I don't know we are specifically deprecating it or not…

The discussion went something along the lines of "request cycle in Bolt\Users === bad… stop using outside classes/things/stuff" and we then moved a lot/most of the permission calls directly to the permission class. 

Knocking it on the head without a clear direction feels a bit wrong to me… so we probably need to step back and look at that… and not in a tiny bug-fix branch… but more than happy to hash it out!